packages/lib/src/vis/tiles/utils.ts

```diff
@@ -50,32 +47,51 @@ export function sortTilesByDistanceTo(
   });
 }
 
-export function getScaledVisibleDomains(
+const NDC_BOX = new Box3(new Vector3(-1, -1, -1), new Vector3(1, 1, 1));
+
+function getObject3DVisibleBox(
+  camera: Camera,
+  context: AxisSystemContextValue,
```

Comment:
This is not used, but I kept it to be compatible with `useCameraState`... but not sure it is useful.

Reply:
Neh, you can remove it, especially with regard to my other comments.